platform-services.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: page
+title: Accessing Platform and Third-Party Services in Flutter
+sidebar: home_sidebar
+permalink: /platform-services/
+---
+
+This guide describes how to connect your Flutter app to platform-specific or
+third-party services that are available on iOS and Android devices.
+
+* TOC
+{:toc}
+
+## Introduction
+
+Mobile devices offer many capabilities to your app which can be accessed through
+platform-specific native APIs, or APIs from third-party vendors. Flutter offers several ways
+for you to integrate the functionality of these of these platform services to extend your Flutter
+app.
+
+### First-party services for Flutter {#first-party-services-for-flutter}
+
+For frequently used services, the Flutter SDK provides APIs to access first-party services that
+are supported across platforms. You can call these APIs directly in your Dart code. These
+first-party services are provided through the `SKY_SERVICES` library. The following list
```

Comment:
`sky_services` (lower case)

s/library/package/

Also, this statement is a bit misleading.  Although its possible to use the mojoms directly, almost everyone should use the more ergonomic APIs in `package:flutter/services` instead.